docs/installation/manual/installing_karaf_and_alpaca.md

```diff
@@ -0,0 +1,346 @@
+Similar to Tomcat, our Karaf service is going to rely on a `setenv` shell script to determine environment variables Karaf needs in place when running. For now, this will simply be the path to `JAVA_HOME`, but this also accepts many other parameters you can find in the default `setenv` script.
+
+`/opt/karaf/bin/setenv | karaf:karaf/755`
+```
+#!/bin/sh
+export JAVA_HOME="PATH_TO_JAVA_HOME"
+```
+- `PATH_TO_JAVA_HOME`: This will be the same `JAVA_HOME` we used when installing Tomcat , and can be found using the same method (i.e., still `/usr/lib/jvm/java-8-openjdk-amd64` if that's what it was before).
+
+### Initializing Karaf
+
+We’re going to start Karaf, then run the installer to put our configurations in place and generate a Karaf service. Once these are installed, we’re going to stop Karaf, as from there on out its start/stop management should be handled via that service.
+
+```bash
+sudo -u karaf /opt/karaf/bin/start
+# You may want to wait a bit for Karaf to start.
+# If you're not sure whether or not it's running, you can always run:
+# ps aux | grep karaf
+# to see if the server is up and running.
+/opt/karaf/bin/client feature:install wrapper
+/opt/karaf/bin/client wrapper:install
+/opt/karaf/bin/stop
+```
+
+### Creating and Starting the Karaf Service
+
+Installing the Karaf wrapper generates several service files that can be used on different types of systems. For this example installation on an Ubuntu 18.04 machine, we want to enable the `karaf.service` service so that Karaf is properly started on boot.
+
+```bash
+sudo systemctl enable /opt/karaf/bin/karaf.service
+sudo systemctl start karaf
+```
+
+## Alpaca 1.0.x
+
+### Adding the Required Karaf Repositories
+
+Karaf features can be installed from several different types of sources, but the fastest and easiest way to do so is from existing repository URLs that we can just plug into Karaf to provide us feature lists prepared and ready for installation. Like most interactions with Karaf, we can add these repositories using its built-in `client`.
+
+!!! notice
+    These repositories are updated consistently, and their updates include revised dependency lists. Commonly, when repositories are out of date or otherwise mismatched, feature installation can result in an `Unable to resolve root: missing requirement` error; for this reason, this guide recommends using recently-updated versions of these repositories. That being said, if such errors occur despite installing the latest versions of these features, the maintainer of the features repository should be informed.
+
+For the Karaf features we’re going to install, we need a few different repositories to be added to the list:
+
+```bash
+/opt/karaf/bin/client repo-add mvn:org.apache.activemq/activemq-karaf/ACTIVEMQ_KARAF_VERSION/xml/features
+/opt/karaf/bin/client repo-add mvn:org.apache.camel.karaf/apache-camel/APACHE_CAMEL_VERSION/xml/features
+/opt/karaf/bin/client repo-add mvn:ca.islandora.alpaca/islandora-karaf/1.0.1/xml/features
```

Comment:
The `1.0.1` is left hardcoded here.  Should push it to a variable like the camel and activemq versions.

Reply:
This also highlights that we're using 1.0.2 on `dev` in islandora-playbook, which we're compiling instead of pulling from an artifact. We should just do another alpaca release so we can bump that up.

Reply:
FYI I just released Alpaca 1.0.2 and it's now available via mvn

Reply:
Gonna move this over to use `LATEST` as well, I think ... not sure I really remember a reason why this was hardcoded

Reply:
For the other ones, it looks like it's going to be safest to hard-code versions when `repo-add`-ing ... looks like at least at the moment for karaf, you end up with some unreleased `SNAPSHOT` builds that don't quite work correctly. Probably going to be safest all around to recommend that level of specificity for repos Islandora isn't in charge of